node/service/src/chain_spec/frequency_rococo.rs

```diff
@@ -53,10 +58,6 @@ pub mod public_testnet_keys {
 		"0x1caccabbe8095a35292782cffc29145030264f9706753923a5c237e8f5aceb1a";
 }
 
-// pub fn load_frequency_rococo_spec() -> Result<ChainSpec, String> {
-// 	ChainSpec::from_json_bytes(&include_bytes!("../../specs/frequency_rococo.json")[..])
-// }
-
 pub fn frequency_rococo_testnet() -> ChainSpec {
```

Comment:
This will be removed in the future, but we want the main commit hash to reference so that people can verify the generated spec file against what generated it.